client/src/app/users/user-list.component.html

```diff
@@ -59,25 +59,25 @@
   </div>
 </div>
 
-<div fxLayout="row">
-  <div fxFlex fxFlex.gt-sm="80" fxFlexOffset.gt-sm="10" *ngIf="serverFilteredUsers; else usersError" >
+<div class="flex-row">
+  <div class="flex-1" fxFlex.gt-sm="80" fxFlexOffset.gt-sm="10" *ngIf="serverFilteredUsers; else usersError" >
```

Comment:
Same here there are attributes for the old flex layout library that don't do anything still here and probably in many other places.